pages/components/button.html

```diff
@@ -0,0 +1,151 @@
+<html lang="en">
+  <%- include('head.html'); -%>
+  <body>
+    <f-docs-template>
+      <%- include('nav.html'); -%>
+      <main slot="content">
+        <h1 class="mb-16">Button</h1>
+        <p>Buttons are used to perform actions, with different visuals for different needs.</p>
+
+        <h2 class="mt-24 mb-16">Props</h2>
+        <table class="w-full docs-table">
+          <thead>
+            <tr class="text-left text-gray-400 uppercase">
+              <th>prop</th>
+              <th>type</th>
+              <th>default</th>
+            </tr>
+          </thead>
+          <tbody class="align-top">
+            <tr>
+              <td>type</td>
+              <td>
+                <div>"button" | "submit" | "reset"</div>
+                <div>Button type, only applied when href is not set.</div>
+              </td>
+              <td>"button"</td>
+            </tr>
+            <tr>
+              <td>variant</td>
+              <td>"primary" | "secondary" | "negative" | "utility" | "link" | "pill"</td>
+              <td>"secondary"</td>
+            </tr>
+            <tr>
+              <td>quiet</td>
+              <td>boolean</td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>small</td>
+              <td>boolean</td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>loading</td>
+              <td>boolean</td>
+              <td>false</td>
+            </tr>
+            <tr>
+              <td>href</td>
+              <td>
+                <div>string</div>
+                <div>Set the href for the location where clicking the button will take you to.</div>
+              </td>
+              <td>-</td>
+            </tr>
+            <tr>
+              <td>target</td>
+              <td>
+                <div>string</div>
+                <div>
+                  Anchor target, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a.
+                </div>
+              </td>
+              <td>-</td>
+            </tr>
+            <tr>
+              <td>rel</td>
+              <td>
+                <div>string</div>
+                <div>The relationship of the linked URL.</div>
+              </td>
+              <td>"noopener"</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h2 class="mt-24">Visual Options</h2>
+
+        <h3 class="pb-16 mt-32">Variants</h3>
+
+        <div id="variantsTable"></div>
+
+        <h3 class="mt-32">Quiet</h3>
+
+        <div class="md:grid md:grid-cols-3">
+          <div class="col-span-2">
+            By default, buttons have a visible stroke and fill. Quiet buttons do not have this
```

Comment:
I don't think it makes sense to copy the variant descriptions over here. We should have one source of truth for these things IMO - so at worst this should somehow be dynamically brought in from the design-docs if we absolutely have to have it here.

Reply:
If the guide of how each variant should be used belongs more to the design documentation, then I think it makes sense to limit the description to the technical minimum here. I updated the descriptions in [f87107c](https://github.com/fabric-ds/elements/pull/95/commits/f87107c614cc9b7ad7edebc7885af824083f0ac5). Wdyt?

Reply:
Looks great!

How much do you think we should be describing each variant?

e.g. https://fabric-tech-docs-rfc.surge.sh/button/

I haven't (yet) described any of the variant/modifier props, as my thinking was the user could go to the design docs to see what those look like and how they should be used.

Do you think we should have some brief description in the tech docs still?

Reply:
Maybe it was mentioned and I failed to understand/remember, but are the docs for vue, react and elements be linked to each component page from the above docs?

If so, I'm trying to picture how the above docs will be used by developers who work with respective frameworks. I have a feeling they would bypass the general tech docs (the above one) and stick to docs of their framework to see how components are used, what other props they accept etc. Then some description, visual presentation of states/variants might be useful, I guess.

Reply:
The goal is for the framework 'docs' to be non-existant, and for each framework to only expose a playground or storybook.

That way we get away from custom texts and sites for each framework, and instead just have component showcases there (which is all the developer should need in combination with the main tech site).

Reply:
I see. How were you planning on presenting props that are not common in all frameworks, like with elements button that support only a number of anchor attributes atm - `target` and `rel`? 

Also, this approach seems to me to make a strong case against using `variant` attribute here instead of respective boolean attributes. Unless there's some way you were thinking to tackle it on one component page 🤔

Reply:
Props aren't presented as universal for frameworks - e.g. on that page there's "Props" under "Vue" - in the case of Button it was just my attempt at a simplification - but I'd be fine not having that table and just having each framework handle its own data.